AElf.Kernel/AccountManager.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AElf.Kernel
+{
+    class AccountManager : IAccountManager
+    {
+        public List<Account> ExecuteTransactionAsync(ITransaction tx)
+        {
+            var accountFrom = ((tx as Transaction).AccountFrom as Account);
+            var accountTo = ((tx as Transaction).AccountFrom as Account);
+            var amount = (tx as Transaction).Amount;
+
```

Comment:
We should avoid the situation where we need to cast, if needed add the properties (as read only) too the interfaces